tests/client.py

```diff
@@ -403,6 +425,501 @@ def get_workflows(self):
+                            "description": "",
+                            "groupName": "",
+                            "displayOrder": 2,
+                            "required": False,
+                            "selectedOptions": [],
+                            "options": [],
+                            "validation": {
+                                "name": "",
+                                "message": "",
+                                "data": "",
+                                "useDefaultBlockList": False
+                            },
+                            "enabled": True,
+                            "hidden": False,
+                            "defaultValue": "",
+                            "isSmartField": False,
+                            "unselectedLabel": "",
+                            "placeholder": ""
+                        }
+                    ],
+                    "default": True,
+                    "isSmartGroup": False
+                }
+            ],
+            "createdAt": 1318534279910,
+            "updatedAt": 1413919291011,
+            "performableHtml": "",
+            "migratedFrom": "ld",
+            "ignoreCurrentValues": False,
+            "metaData": [],
+            "deletable": True
+        }
+        
+        # generate a record
+        response = self.post(url, data)
+        records = [response]
+        return records
+
+    def create_owners(self):
+        """
+        HubSpot API The Owners API is read-only. Owners can only be created in HubSpot.
+        TODO - use selenium
+        """
+        raise NotImplementedError("Only able to create owners from web app")        
+        
+    def create_subscription_changes(self, subscription_id=''):
+        """
+        HubSpot API https://legacydocs.hubspot.com/docs/methods/email/update_status
+        This will update email_events as well.
+        TODO For updating sub_changes, utilize sub_id as an arg and make a passthrough method
+        """
+        record_uuid = str(uuid.uuid4()).replace('-', '')
+        subscriptions = self.get_subscription_changes()
+        subscription_id_list = [[change.get('subscriptionId') for change in subscription['changes']] for subscription in subscriptions]
+        
+        a_sub_id =random.choice([item[0] for item in subscription_id_list if item[0]])
+        
+        url = f"{BASE_URL}/email/public/v1/subscriptions/{{}}"
+        data = {
+            "subscriptionStatuses": [
+                {
+                    "id": a_sub_id,
+                    "subscribed": True,
+                    "optState": "OPT_IN",
+                    "legalBasis": "PERFORMANCE_OF_CONTRACT",
+                    "legalBasisExplanation": "We need to send them these emails as part of our agreement with them."
+                }
+            ]
+        }
+        
+        # generate a record
+        response = self.put(url.format(record_uuid+"@stitchdata.com"), data)
```

Comment:
Feel free to ignore this, it's nit-picky.

> url.format(record_uuid+"@stitchdata.com"

I don't like this `format()` here. We already know `record_uuid` by the time we initialize `url`.

So I think this should be done on line 860 instead